run.py

```diff
@@ -171,6 +171,8 @@ def main(args):
   # Only import numenta detector if used so as to avoid unnecessary dependency
   if "numenta" in args.detectors:
     from nab.detectors.numenta.numenta_detector import NumentaDetector
+  if "numentaNew" in args.detectors:
```

Comment:
I don't think numentaNew is a good name for this detector. What would we call the next version, numentaReallyNew? Something more specific would be better, such as numentaTM - any suggestions?